app/bedingungen/page.js

```diff
@@ -0,0 +1,20 @@
+import getContentItem from "../../lib/getContentItem";
+import Imprint from "./bedingungen.js";
+
+export const config = {
```

Comment:
Do we still need the `unstable_excludeFiles` configuration. If I remember correctly it was because of some build problems. I guess if the site builds fine without we can remove it.